examples/README.md

```diff
@@ -76,16 +76,19 @@ To run it in each of these various modes, use the following commands:
         ```
     * With PyTorch launcher only (`torch.distributed.launch` can be used in older versions of PyTorch)
         ```bash
-        python -m torchrun --nproc_per_node 2 \
-            --use_env \
-            --node_rank 0 \
-            --master_addr master_node_ip_address \
-            ./nlp_example.py  # On the first server
-        python -m torchrun --nproc_per_node 2 \
-            --use_env \
-            --node_rank 1 \
-            --master_addr master_node_ip_address \
-            ./nlp_example.py  # On the second server
+        torchrun --nproc_per_node 2 \
+            --nnodes 2
+            --rdzv_id 2299 \ # A unique job id 
+            --rdzv_backend c10d \
+            --rdzv_endpoint master_node_ip_address:29500 \
+            ./nlp_example.py # On the first server
+
+        torchrun --nproc_per_node 2 \
```

Comment:
This is the exact same command, no? Is the rank determined automatically or is it assumed that the corresponding env var is set? Either way, I would just say something like: "Run the same command on the 2nd node, with the rank ...".

Reply:
Correct, there is no need to specify the rank, and the command is the same on both nodes. Initially, I set it up similarly to how it was before, but we can implement the suggestion you mentioned.

Reply:
In that case, my personal preference would be to say: "Run this command on each node" and avoid the repetition. @muellerzr what do you prefer?

Reply:
TIL we no longer need to specify the rank. That's something we can probably change in `accelerate config` to speed things up if it's truly unnecessary (or at least in the `accelerate launch` examples). Let's do "Run this command on each node" then